src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderControllerTest.java

```diff
@@ -64,27 +70,37 @@ void beforeEach() {
     @DisplayName("Normal service call returns response")
     @Test
     void callReturnsResponse() {
-        final ConvictionResponse attendancesResponse = ConvictionResponse.builder().attendances(Collections.emptyList()).build();
-        when(convictionService.getConviction(CRN, SOME_EVENT_ID)).thenReturn(attendancesResponse);
+        final SentenceResponse attendancesResponse = SentenceResponse.builder().attendances(Collections.emptyList()).build();
+        when(convictionService.getSentence(CRN, SOME_EVENT_ID, SOME_SENTENCE_ID)).thenReturn(attendancesResponse);
 
-        assertThat(controller.getConviction(CRN, SOME_EVENT_ID)).isEqualTo(attendancesResponse);
+        assertThat(controller.getSentence(CRN, SOME_EVENT_ID, SOME_SENTENCE_ID)).isEqualTo(attendancesResponse);
 
-        verify(convictionService).getConviction(CRN, SOME_EVENT_ID);
+        verify(convictionService).getSentence(CRN, SOME_EVENT_ID, SOME_SENTENCE_ID);
         verifyNoMoreInteractions(convictionService);
     }
 
-    @DisplayName("Feature toggle is off")
+    @DisplayName("Feature toggle for sentence data is off")
     @Test
-    void featureToggleFalse() {
-        featureFlags.setFlagValue("fetch-attendance-data", false);
-        final ConvictionResponse convictionResponse = ConvictionResponse.builder()
+    void featureToggleFalseSentenceData() {
+        featureFlags.setFlagValue("fetch-sentence-data", false);
+        final SentenceResponse sentenceResponse = SentenceResponse.builder()
                 .attendances(Collections.emptyList())
                 .unpaidWork(UnpaidWork.builder().build())
+                .currentOrderHeaderDetail(CurrentOrderHeaderResponse.builder().custodialType(KeyValue.builder().code(null).description(null).build())
+                        .sentenceDescription(null)
```

Comment:
might be easier if we just didn't call all these will null. They'll be null without that.

Reply:
Good spot